projects/usrsctp/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/sctplab/usrsctp"
+primary_contact: "weinrank@fh-muenster.de"
+auto_ccs:
+  - "tuexen@fh-muenster.de"
+sanitizers:
+  - address
+  - memory:
+    experimental: True
```

Comment:
Please remove the experimental flag here. This should not be needed unless your library has system dependencies and which is not in our instrumented list.

Reply:
Done!